build-targets/steelseries-rival310.txt

```diff
@@ -1,2 +1,4 @@
 [properties]
-linker = 'stm32f103x6'
+target = 'rival310'
+linker = 'rival310-oem-boot'
+sensor = 'truemove3'
```

Comment:
Shouldn't this be better in the `.h`?

Reply:
yes, this is just here while i don't implement the config file.